tests/invite-api.js

```diff
@@ -127,10 +136,14 @@ test('Reject invite', async (t) => {
   r1.on('peers', async (peers) => {
     t.is(peers.length, 1)
 
-    const response = await r1.invite(peers[0].deviceId, {
+    const invite = {
       projectKey,
       encryptionKeys,
-    })
+      projectInfo: { name: 'Mapeo' },
+      roleName: DEFAULT_CAPABILITIES[MEMBER_ROLE_ID].name,
```

Comment:
missing `roleId`

Reply:
same as comment avobe. The `invite` here refers to the protobuf, so `roleId` is not needed